core-bundle/src/Twig/Runtime/CspRuntime.php

```diff
@@ -51,4 +51,17 @@ public function addSource(string $directive, string $source): void
         $csp = $responseContext->get(CspHandler::class);
         $csp->addSource($directive, $source);
     }
+
+    public function addHash(string $directive, string $source, string $algorithm = 'sha384'): void
```

Comment:
Is it really necessary to use `sha384` here? A faster algorithm would probably suffice and would be, well, faster. 😄

Reply:
I‘m going to create separate issue for that.